README.md

```diff
@@ -129,15 +130,49 @@ cargo fmt --all
 cargo clippy --workspace --all-features -- -D warnings
 ```
 
+## Release Process
+
+### Overview
+
+We make releases within the `rust` repository specific to the crate(s) that will be consumed (e.g. agents@1.0.0, configuration@1.0.0, accumulator@1.0.0, etc).
+
+We follow [Semantic Versioning](https://semver.org/), where breaking changes constitute changes that break agent configuration compatibility.
+
+Releases are managed on GitHub [here](https://github.com/nomad-xyz/rust/releases).
+
+### Aggregating Release Notes
+
+- Want to aggregate list of all changes since last release
+- Run `git diff <tag of last release> HEAD -- **CHANGELOG.md`
+- Manually consolidate diffs into a single list for release notes
+
+### Bumping Versions
+
+- Bump package versions in all relevant `Cargo.toml` files
+- Bump the package versions in all relevant `CHANGELOG.md` files
+- E.g. for an `agents` release, this would entail bumping all agents in `rust/agents`
+- Make/merge a PR declaring the new version you are releasing (e.g. "Bumping agents for release agents@1.0.1")
+
+### Making a New Release
+
+- Tag newly-merged PR by running `git tag -s <package(s)-release-name>@<new-package-version>` (e.g. `git tag -s agents@1.0.1`)
```

Comment:
tag description should contain the changes from the changelog patch above?

Reply:
Yes probably, will add